src/main/utils/goUtils.ts

```diff
@@ -28,6 +28,8 @@ export class GoUtils {
             .toString()
```

Comment:
Let's add tests for getGoVersion checking the results with and without the patch version

Reply:
it will be difficult since the go version returned is the version installed.
I added a test running this once for the current version the machine uses